### PR TITLE
ci: add action workflow to valid pull request for ui/web-v2

### PR DESCRIPTION
**Pull request by cre8ivejp**
The current CI for the `ui/web-v2` pull request runs on the Argo workflow. Because the CI website is for internal use only, I'm migrating it from Argo to GitHub Actions so everyone can see the result logs.

Results from the action workflow.
https://github.com/bucketeer-io/bucketeer/actions/runs/3172891781

### CODE REVIEW

File: ui/web-v2/Makefile [hunk 1]
```diff
@@ -1,3 +1,6 @@
+.PHONY: all
+all: install buildifier-check lint gen_proto build
```
Comment (cre8ivejp) [inline ui/web-v2/Makefile:2]: I added this so people can check everything locally before opening the PR.

File: .github/workflows/pr-ui-web.yaml [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  pull_request:
+    branches:
+      - main
+    paths:
+      - "ui/web-v2/**"
+      - "!**/**.md"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
```
Comment (cre8ivejp) [inline .github/workflows/pr-ui-web.yaml:13]: This setting will cancel the current CI if you push something while the CI is still running.

File: .github/workflows/pr-ui-web.yaml [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+      - uses: actions/cache@v3
+        id: yarn-cache
+        with:
+          path: |
+            **/node_modules
+            **/.eslintcache
+            ${{ steps.yarn-cache-dir-path.outputs.dir }}
```
Comment (masaaania) [inline .github/workflows/pr-ui-web.yaml:38]: You may have forgotten to add a step whose id is `yarn-cache-dir-path`.

File: .github/workflows/pr-ui-web.yaml [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
```
Comment (masaaania) [inline .github/workflows/pr-ui-web.yaml:62]: Same as above.

File: .github/workflows/pr-ui-web.yaml [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
```
Comment (masaaania) [inline .github/workflows/pr-ui-web.yaml:86]: Same as above.

File: ui/web-v2/Makefile (hunk 1 above)
Comment (masaaania) [reply ui/web-v2/Makefile:2]: Thank you! 😄

File: .github/workflows/pr-ui-web.yaml (hunk 3 above)
Comment (cre8ivejp) [reply .github/workflows/pr-ui-web.yaml:41]: @masaaania good catch, thanks!
I have fixed it.
You can see the results here:
https://github.com/bucketeer-io/bucketeer/actions/runs/3179533843

Comment (masaaania) [reply .github/workflows/pr-ui-web.yaml:41]: @cre8ivejp 

Looks good to me about `pr-ui-web.yaml`!!
Are you middle in work about `pr-go.yaml`?

Comment (cre8ivejp) [reply .github/workflows/pr-ui-web.yaml:41]: I pushed it by accident.
I will delete it, thanks!

Comment (cre8ivejp) [reply .github/workflows/pr-ui-web.yaml:41]: @masaaania 
I have deleted it. Please take a look at it tomorrow.

**Review by masaaania — APPROVED** [review]
LGTM👍 
Thank you!

**Pull request merged**